docs/spec_draft.md

```diff
@@ -1267,6 +1268,137 @@ More formally, `result[i0, ..., iR-1]` is equal to:
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.reduce_window
+
+### Semantics
+
+Performs a reduction function `body` over all elements in each window of a
+sequence of N `inputs` tensors and produces a sequence of N `results` tensor.
+
+The following diagram shows how elements in `results[k]` are computed from
+`operand[k]` using a concrete example.
+
+<p align="center">
+  <img src="spec_draft/reduce_window.png" />
+</p>
+
+Conceptually, the kth dimension of `inputs[k]` is dilated by implicitly
+placing `base_dilations[k] - 1` holes between each of the entries in that
+dimension.
+
+`padding[k][0]` and `padding[k][1]` specify the amount of
```

Comment:
This should be updated to match 1-dimensional `padding`.

Reply:
Please see the comment above.